### PR TITLE
Added: Get album art from Cover Art Archive

### PR DESCRIPTION
**Pull request by danielunderwood**
For #78.

### CODE REVIEW

**Review by Qstick — COMMENTED** [review]
Is this setup to fall back to fanart currently?

File: lidarrmetadata/provider.py [hunk 1]
```diff
@@ -524,7 +525,26 @@ def get_artist_by_id(self, artist_id):
...
+        art = {}
+        for result in results:
+            cover_type = type_mapping.get(result['type'], None)
+            if cover_type is not None and cover_type not in art:
+                art[cover_type] = self._build_caa_url(result['release_gid'], result['type'])
+        return [{'CoverType': art_type, 'Url': url} for art_type, url in art.items()]
+
+    @staticmethod
+    def _build_caa_url(release_id, art_type):
+        return 'https://coverartarchive.org/release/{}/{}.jpg'.format(release_id, art_type.lower())
```
Comment (Qstick) [inline lidarrmetadata/provider.py]: Images are not stored by art type name. They appear to be stored by ID. 

Ex. http://coverartarchive.org/release/76df3287-6cda-33eb-8e9a-044b5e15ffdd/829521842.jpg

Comment (danielunderwood) [reply lidarrmetadata/provider.py]: Hopefully resolved by 89a0480.

**Comment by danielunderwood** [comment]
I have added a quick fallback to fanart in 265b3ac, but I'll have to do a bit of thinking to find a good way to provide it across sources so we can do things like #116 cleanly.

**Review by Qstick — APPROVED** [review]
I'm good unless there was something else you wanted to add.

**Pull request merged**